definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "chrisamiller/homer"
+    - class: ResourceRequirement
+      ramMin: 32000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'homer_tag_directory.sh'
+        entry: |
+		set -o pipefail
+		set -o errexit
+
+		name=$1
+		bam=$2
+		reference_fasta=$3  #assumes that there's a .genome file next to the reference .fa
+		#run in chrisamiller/homer container
+		export PATH=$PATH:/opt/homer/bin
+		echo "name:     $name"
+		echo "bam:      $bam"
+		echo "reffasta: $reference_fasta"
+		if [[ ! -d $name/homer ]];then
+  			mkdir -p $name/homer
+		fi
+		echo "creating tagDir"
+		makeTagDirectory $name/homer $bam
+		echo "tagdir2Bed"
+		/gscuser/dspencer/programs/homer/bin/tagDir2bed.pl $name/homer/ | awk '$2>0 && $3>$2' | awk -v n=$name '{print >n"/"n".bed.part."$1}'
```

Comment:
This tagdir to bed stuff can all be dropped, AFAIK (lines 32-38). As long as we have the tagdir, we can run the downstream homer peak calling and such. I don't think I've ever actually used the tagdir bed file for anything